apps/meteor/app/api/server/v1/rooms.ts

```diff
@@ -140,6 +140,7 @@ API.v1.addRoute(
 	},
 );
 
+// TODO: deprecate API
```

Comment:
Please undo this change, it's from my previous PR

Reply:
Done